NPE Log Spam From MessageEntitySync Packet

You aren't null checking the entity after retrieving it from the world using the ID passed in the packet. This is resulting in log spam of NPE's in my console... I have Tinkers' Construct installed so maybe their slimes are causing issues? I'm not sure.

**Mod version:** [1.0.0](https://minecraft.curseforge.com/projects/slimyboyos/files/2500372)
**Forge version:** [1.12.2-14.23.2.2623](http://files.minecraftforge.net/maven/net/minecraftforge/forge/1.12.2-14.23.2.2623/forge-1.12.2-14.23.2.2623-changelog.txt)

**Relevant line:** [MessageEntitySync.java#L50](https://github.com/jaredlll08/SlimyBoyos/blob/master/src/main/java/com/blamejared/slimyboyos/network/MessageEntitySync.java#L50)
**Stacktrace:**

```
[main/FATAL]: Error executing task
java.util.concurrent.ExecutionException: java.lang.NullPointerException
	at java.util.concurrent.FutureTask.report(Unknown Source) ~[?:1.8.0_144]
	at java.util.concurrent.FutureTask.get(Unknown Source) ~[?:1.8.0_144]
	at net.minecraft.util.Util.func_181617_a(SourceFile:47) [h.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1086) [bib.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:397) [bib.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:123) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_144]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_144]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_144]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_144]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
Caused by: java.lang.NullPointerException
	at com.blamejared.slimyboyos.network.MessageEntitySync.handle(MessageEntitySync.java:50) ~[MessageEntitySync.class:?]
	at com.blamejared.slimyboyos.network.MessageEntitySync.lambda$onMessage$0(MessageEntitySync.java:42) ~[MessageEntitySync.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[?:1.8.0_144]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[?:1.8.0_144]
	at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
	... 9 more
```

--- a/src/main/java/com/blamejared/slimyboyos/network/MessageEntitySync.java
+++ b/src/main/java/com/blamejared/slimyboyos/network/MessageEntitySync.java
@@ -47,7 +47,8 @@ public class MessageEntitySync implements IMessage, IMessageHandler<MessageEntit
     private void handle(MessageEntitySync message, MessageContext ctx) {
         if(FMLClientHandler.instance().getClient().world != null) {
             Entity entityByID = FMLClientHandler.instance().getClient().world.getEntityByID(message.id);
-            entityByID.setItemStackToSlot(EntityEquipmentSlot.HEAD, message.headStack);
+            if (entityByID != null)
+                entityByID.setItemStackToSlot(EntityEquipmentSlot.HEAD, message.headStack);
         }
     }
     

